Formula/opentracing-cpp.rb

```diff
@@ -0,0 +1,146 @@
+class OpentracingCpp < Formula
+  desc "OpenTracing API for C++"
+  homepage "http://opentracing.io"
+  url "https://github.com/opentracing/opentracing-cpp/archive/v1.5.0.tar.gz"
+  sha256 "4455ca507936bc4b658ded10a90d8ebbbd61c58f06207be565a4ffdc885687b5"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-DCMAKE_INSTALL_PREFIX=#{prefix}", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    # This is from example/tutorial/tutorial-example.cpp, minus empty lines
```

Comment:
We should do `pkgshare.install "example/tutorial/tutorial-example.cpp"` in the install block, so we can then use that file directly instead of copying it in the formula.